app/models/concerns/acts_as_tagged.rb

```diff
@@ -0,0 +1,41 @@
+module ActsAsTagged
+  extend ActiveSupport::Concern
+
+  module Base
+    def acts_as_tagged(*options)
+      @acts_as_tagged = options || []
+      include ::ActsAsTagged
+    end
+  end
+
+  included do
+    has_many :taggings, as: :taggable, class_name: 'Effective::Tagging', dependent: :delete_all
+    has_many :tags, through: :taggings, class_name: 'Effective::Tag'
+
+    accepts_nested_attributes_for :tags, reject_if: :all_blank, allow_destroy: true
+
+    def tags=(ids)
```

Comment:
How can we use this to also create new tags?

Reply:
Gotcha, sorry,  I thought those titles were some grouping that tags belonged to, I copied that code and is working great, thanks Matt!